src/sqlancer/cockroachdb/CockroachDBToStringVisitor.java

```diff
@@ -231,4 +249,155 @@ public void visit(CockroachDBMultiValuedComparison comp) {
         sb.append(")");
         sb.append(")");
     }
+
+
+    @Override
+    public void visit(CockroachDBExists existsExpr) {
+        if (existsExpr.getNegated()) {
+            sb.append(" NOT");
+        }
+        sb.append(" EXISTS(");
+        visit(existsExpr.getExpression());
+        sb.append(")");
+    }
+
+    @Override
+    public void visit(CockroachDBExpressionBag exprBag) {
+        visit(exprBag.getInnerExpr());
+    }
+
+    @Override
+    public void visit(CockroachDBValues values) {
+        LinkedHashMap<CockroachDBColumn, List<CockroachDBConstant>> vs = values.getValues();
+        int size = vs.values().iterator().next().size();
+        sb.append("(VALUES ");
+        for (int i = 0; i < size; i++) {
+            sb.append("(");
+            for (CockroachDBColumn c: vs.keySet()) {
+                sb.append(vs.get(c).get(i).toString());
+                if (!(vs.get(c).get(i) instanceof CockroachDBNullConstant)) {
+                    if (c.getType() != null) {
+                        sb.append("::" + c.getType().toString());
+                    }
+                }
+                sb.append(", ");
+            }
+            sb.deleteCharAt(sb.length() - 1);
```

Comment:
Deleting previous parts seems quite complicated. Can we simplify this (and similar logic in other files)?